include/cantera/thermo/IdealSolidSolnPhaseTabulatedThermo.h

```diff
@@ -0,0 +1,135 @@
+/*
+ * IdealSolidSolnPhaseTabulatedThermo.h
+ *
+ *  Created on: 04.06.2016
+ *      Author: mmayur
+ */
+
+/**
+ *  @file IdealSolidSolnPhaseTabulatedThermo.h
+ * Definition file for a derived class of ThermoPhase that assumes either
+ * an ideal gas or ideal solution approximation and handles

```

Comment:
If this is for an ideal solid, the documentation should be updated to reflect that it is not a gas